lib/ansible/modules/network/nxos/nxos_file_copy.py

```diff
@@ -206,46 +205,44 @@ def main():
     if not HAS_SCP:
         module.fail_json(
             msg='library scp is required but does not appear to be '
-                'installed.  It can be installed using `pip install scp`'
-            )
+                'installed. It can be installed using `pip install scp`'
+        )
 
     warnings = list()
     check_args(module, warnings)
 
+    results = dict(changed=False, warnings=warnings)
+
     local_file = module.params['local_file']
     remote_file = module.params['remote_file']
     file_system = module.params['file_system']
 
-    changed = False
-    transfer_status = 'No Transfer'
+    results['transfer_status'] = 'No Transfer'
+    results['local_file'] = local_file
+    results['file_system'] = file_system
+    results['changed'] = False
```

Comment:
I believe this is over initialization as it is already `False` by default `results = dict(changed=False, warnings=warnings)`.